sample/Sample.FunctionalTests/Sample.FunctionalTests.csproj

```diff
@@ -19,7 +19,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\SampleEndpointApp\SampleEndpointApp.csproj" />
+      <ProjectReference Include="..\..\src\Ardalis.ApiEndpoints\Ardalis.ApiEndpoints.csproj">
+          <SetTargetFramework>TargetFramework=netstandard2.1</SetTargetFramework>
```

Comment:
It doesn't use `netstandard2.1` by default in `net6`? What does it use? Will this be an issue for NuGet consumers?

Reply:
Maybe it makes sense to migrate whole package to `netstandard2.1`. It's only intended to be used in ASP.NET Core anyway

Reply:
> Will this be an issue for NuGet consumers?
I remember reading something about it, but did not have time to try.


> Maybe it makes sense to migrate whole package to `netstandard2.1`. It's only intended to be used in ASP.NET Core anyway
I would personally like to upgrade to netstandard2.1.
The compatibility table is here:
https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1#select-net-standard-version

Reply:
@ardalis How about updating to netstandard2.1?

Reply:
@ardalis any thoughts about upgrading to netstandard2.1?

Reply:
Yes that's fine with me.

Reply:
#180 has bumped the target framework to netcore3.1, so I removed the netstandard2.1 stuff.
code is so much cleaner now :)